MapboxNavigationUI/MGLMapView+MGLNavigationAdditions.h

```diff
@@ -0,0 +1,7 @@
+#import <Mapbox/Mapbox.h>
+
+@interface MGLMapView (MGLNavigationAdditions)
+
+- (void)locationManager:(CLLocationManager *)manager didUpdateLocations:(NSArray *)locations;
```

Comment:
Whichever method we choose to override, I'm okay with doing this logic here for the time being while we experiment.